docs/camino-node/azure.md

```diff
@@ -0,0 +1,278 @@
+This will make your node accessible to anyone on the internet.
+
+:::
+
+<figure>
+<img class="zoom" src="/img/azure/azure_vm_13.png#center"/>
+<figcaption align="center"><b>Fig.13:</b> Allow SSH port</figcaption>
+</figure>
+
+After setting up the security rules, click **Ok**.
+
+<figure>
+<img class="zoom" src="/img/azure/azure_vm_14.png#center"/>
+<figcaption align="center"><b>Fig.14</b></figcaption>
+</figure>
+
+### 9. Inserting Custom Script
+
+- Navigate to the **Advanced** tab.
+
+<figure>
+<img class="zoom" src="/img/azure/azure_vm_15.png#center"/>
+<figcaption align="center"><b>Fig.15</b></figcaption>
+</figure>
+
+- Under **Custom data**, input the script below.
+
+<figure>
+<img class="zoom" src="/img/azure/azure_vm_16.png#center"/>
+<figcaption align="center"><b>Fig.16</b></figcaption>
+</figure>
+
+```bash
+#!/bin/bash
+
+apt-get update
+apt-get install -y ca-certificates curl gnupg
+
+install -m 0755 -d /etc/apt/keyrings
+curl -fsSL https://download.docker.com/linux/ubuntu/gpg | gpg --dearmor -o /etc/apt/keyrings/docker.gpg
+chmod a+r /etc/apt/keyrings/docker.gpg
+
+echo \
+  "deb [arch="$(dpkg --print-architecture)" signed-by=/etc/apt/keyrings/docker.gpg] https://download.docker.com/linux/ubuntu \
+  "$(. /etc/os-release && echo "$VERSION_CODENAME")" stable" | \
+  tee /etc/apt/sources.list.d/docker.list > /dev/null
+
+apt-get update
+apt-get install -y docker-ce docker-ce-cli containerd.io docker-buildx-plugin docker-compose-plugin
+
+apt-get install -y apt-transport-https ca-certificates gnupg curl sudo
+
+echo "deb [signed-by=/usr/share/keyrings/cloud.google.gpg] https://packages.cloud.google.com/apt cloud-sdk main" | tee -a /etc/apt/sources.list.d/google-cloud-sdk.list
+
+curl https://packages.cloud.google.com/apt/doc/apt-key.gpg | apt-key --keyring /usr/share/keyrings/cloud.google.gpg add -
+
+apt-get update && apt-get install -y google-cloud-cli
+
+mkdir -p /home/camino-data/db
+## for columbus use gs://columbus-db
+gsutil -m rsync -d -R  gs://camino-db /home/camino-data/db
+
+docker run -d -v /home/camino-data:/root/.caminogo -p 9650:9650 -p 9651:9651 --restart always c4tplatform/camino-node:latest ./camino-node --network-id=camino --http-host=0.0.0.0 --public-ip-resolution-service=ifconfigme
+
+```
+
+### 10. Adding Tags
+
+- Switch to the **Tags** tab.
+- Add relevant tags to your virtual machine, such as `camino-node` or `network-id: columbus`.
+
+<figure>
+<img class="zoom" src="/img/azure/azure_vm_18.png#center"/>
+<figcaption align="center"><b>Fig.18</b></figcaption>
+</figure>
+
+### 11. Finalization
+
+- Click **Review and Create**.
+- Finally, select **Create** to deploy your VM.
+
+<figure>
+<img class="zoom" src="/img/azure/azure_vm_19.png#center"/>
+<figcaption align="center"><b>Fig.19</b></figcaption>
+</figure>
+
+- Don't forget to download the public key file to securely connect to your virtual machine in the future.
```

Comment:
There is a screenshot here to be added (azure_vm_20.png)
as azure after clicking create it pop up a window to download the private key

Reply:
Fixed